setup.py

```diff
@@ -534,8 +582,7 @@ def pxd(name):
         'depends': _pxi_dep['sparse']},
     '_libs.tslib': {
```

Comment:
For demonstration purposes: note that util has been removed entirely from the pxdfiles entries for tslib/tslibs (as opposed to changed to _libs/util).  If you go and edit tslibs/util.pxd and re-run build_ext, you'll messages to the effect of:

```
Compiling pandas/_libs/tslibs/resolution.pyx because it depends on ./pandas/_libs/tslibs/util.pxd.
Compiling pandas/_libs/algos.pyx because it depends on ./pandas/_libs/tslibs/util.pxd.
Compiling pandas/_libs/tslibs/nattype.pyx because it depends on ./pandas/_libs/tslibs/util.pxd.
[...]
```